package.json

```diff
@@ -29,6 +29,7 @@
     "zone.js": "^0.8.14"
   },
   "devDependencies": {
+    "@angular-devkit/core": "0.0.29",
```

Comment:
why do we need this?

Reply:
There is issue while doing `ng serve`, I managed to find better solution.